idaes/core/surrogate/pysmo_surrogate.py

```diff
@@ -328,6 +328,22 @@ def _create_model(self, pysmo_input, output_label):
     def _get_metrics(self, model):
         return {"RMSE": model.errors["MSE"] ** 0.5, "R2": model.errors["R2"]}
 
+    def get_confidence_intervals(self, model, confidence=0.95):
+        """
+        Compute confidence intervals for the regression patamaters.
+
+        Args:
+            model           : Trained PysmoPolyTrainer object
+            confidence      : Required confidence interval level, default = 0.95 (95%)
+
+        Returns:
+            dict(<dict>)    : Dictionary object containing confidence intervals for all regressed parameters.
```

Comment:
what are the keys and what are the values?

Reply:
Output type indicated.

The keys are the list of output variables which the user supplied, so it should be fairly straightforward. I'll add a sentence here though.

Reply:
Done.